datasets/PV24_flevoland/graph_values.yml

```diff
@@ -122,67 +122,67 @@ transport_shipping_mixer_diesel:
     :biodiesel: 0.0
     :heavy_fuel_oil: 0.0
 transport_final_demand_bio_ethanol:
-  demand: 449.371454
+  demand: 438.382216
 transport_final_demand_coal:
   demand: 0.0
 transport_rail_mixer_diesel:
   input:
-    :diesel: 0.9727388572565022
-    :biodiesel: 0.027261142743497793
+    :diesel: 0.9292625241883464
+    :biodiesel: 0.0707374758116535
 transport_final_demand_biodiesel:
-  demand: 241.628546
+  demand: 420.617784
 transport_final_demand_for_shipping_lng:
   demand: 0.0
 transport_useful_demand_bicycles:
-  demand: 347.42888167928567
+  demand: 349.49010536822857
 transport_final_demand_heavy_fuel_oil:
   demand: 0.0
 transport_final_demand_electricity:
-  demand: 306.003548
+  demand: 464.54743199999996
 transport_final_demand_kerosene:
   demand: 0.0
 energy_power_ultra_supercritical_network_gas:
   demand: 0.0
-  full_load_hours: 4600.0
+  full_load_hours: 1590.0
```

Comment:
What's behind this significant decrease in full load hours? Is this due to the national update from 2018 to 2019?

Reply:
This is correct looking at the source-data and directly derived from the 2019 NL dataset. @mabijkerk do you have any idea?

Reply:
According to @mabijkerk: The installed capcitiy of gas powerplants has remained the same but gas powerplants have produced less electricity in 2019 compared to 2015 which leads to a decrease in full load hours.